src/controllers/update/remove.confirm.statement.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { NextFunction, Request, Response } from "express";
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+import { ApplicationData } from "../../model/application.model";
+import { getApplicationData } from "../../utils/application.data";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.REMOVE_CONFIRM_STATEMENT_PAGE}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+    return res.render(config.REMOVE_CONFIRM_STATEMENT_PAGE, {
+      backLinkUrl: `${config.UPDATE_REGISTRABLE_BENEFICIAL_OWNER_URL}`,
+      appData
+    });
+  } catch (error) {
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.REMOVE_CONFIRM_STATEMENT_PAGE}`);
+    return res.redirect(`${config.REMOVE_CANNOT_USE_URL}`);
```

Comment:
Is this redirection just a temporary implementation? Until the next screen/story is done? If yes, might be worth making this clear with a TODO comment.

Reply:
Added TODO